packages/core/src/xml-transformations/xslt-lowering-writer.ts

```diff
@@ -0,0 +1,307 @@
+import * as fileSystem from "fs";
+import * as path from "path";
+import {OutputStream} from "../io/stream/output-stream";
+
+import {
+  XmlTransformation,
+  XmlTemplate,
+  xmlMatchIsLiteral,
+  xmlMatchIsClass,
+  XmlRootTemplate,
+  XmlMatch,
+} from "./xslt-model";
+
+import { XmlWriter, XmlStreamWriter } from "../xml-schema/xml-writer";
+
+import { XSLT_LOWERING } from "./xslt-vocabulary";
+
+const xslNamespace = "http://www.w3.org/1999/XSL/Transform";
+
+export async function saveXsltLoweringToDirectory(
```

Comment:
I know we use to have this method, but we should remove it. The writers should not access FileSystem directly.

Reply:
This function might not be needed anymore; I have copied it over from the XSD generator, so I can remove them from both generators eventually.